tests/Integration/PrestaShopBundle/Command/LoadLegacyClassesinCommandTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace Tests\Integration\PrestaShopBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class LoadLegacyClassesinCommandTest extends KernelTestCase
+{
+    private $previousErrorReportingLevel;
+
+    public function setUp()
+    {
+        parent::setUp();
+        self::bootKernel();
+        $this->previousErrorReportingLevel = error_reporting(E_WARNING);
+    }
+
+    public function testLoadLegacyCommandWithoutContextFails()
+    {
+        $this->expectException(\TypeError::class, 'Not enough arguments (missing: "theme, locale").');
```

Comment:
```suggestion
        $this->expectException(\TypeError::class, 'Not enough arguments (missing: "theme, locale").');d
```

define `TypeError` in use.